src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/NewValueTupleExpressionBuilderContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Riok.Mapperly.Descriptors.Mappings;
+using Riok.Mapperly.Descriptors.Mappings.MemberMappings;
+
+namespace Riok.Mapperly.Descriptors.MappingBodyBuilders.BuilderContext;
+
+/// <summary>
+/// An implementation of <see cref="INewValueTupleBuilderContext{T}"/>.
+/// </summary>
+/// <typeparam name="T">The type of the mapping.</typeparam>
+public class NewValueTupleExpressionBuilderContext<T> : MembersContainerBuilderContext<T>, INewValueTupleBuilderContext<T>
+    where T : INewValueTupleMapping, IMemberAssignmentTypeMapping
+{
+    public NewValueTupleExpressionBuilderContext(MappingBuilderContext builderContext, T mapping)
+        : base(builderContext, mapping) { }
+
+    public void AddTupleConstructorParameterMapping(ValueTupleConstructorParameterMapping mapping)
+    {
+        if (MemberConfigsByRootTargetName.TryGetValue(mapping.Parameter.Name, out var value))
+        {
+            value.RemoveAll(x => x.Target.Path.Count == 1);
```

Comment:
If `value` is empty after removing these values, we should remove it from `MemberConfigsByRootTargetName` to make the code at other locations easier.